### PR TITLE
feat: List new farms and pools

### PR DESCRIPTION
**Pull request by ChefMomota**
---

## PR-Codex overview
This PR adds new pools, farms, tokens, and gauges related configurations for PancakeSwap on BSC and Ethereum chains.

### Detailed summary
- Added a new pool with `sousId: 374` and `stakingToken: cake`
- Added new farms with `pid: 155` and `lpAddress: 0x004A335932D50f666011935677d96a366AF3eAe2`
- Introduced new tokens like `chat`, `mb4`, `mc`, `octavia`
- Included new gauges configurations for various pairs on BSC and Ethereum chains

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by chefjackson** [comment]
@ChefMomota could you help add changeset as well?

**Comment by ChefMomota** [comment]
done

File: .changeset/nice-dolphins-marry.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+'@pancakeswap/gauges': major
+'@pancakeswap/farms': major
+'@pancakeswap/pools': major
+'@pancakeswap/tokens': minor
+---
+
+List new pool, farm, token and gauges
```
Comment (chefjackson) [inline .changeset/nice-dolphins-marry.md:8]: Should be patch/minor instead of major

Comment (chefjackson) [reply .changeset/nice-dolphins-marry.md:8]: let's use patch

**Review by chefjackson — APPROVED** [review]

**Pull request merged**